company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -77,7 +80,7 @@ public void totalOrderValuesByCityUsingPrimitiveValues()
     {
         Function<Customer, String> cityFunction = Customer::getCity;
         DoubleFunction<Customer> totalOrderValueFunction = Customer::getTotalOrderValue;
-        ObjectDoubleMap<String> map = null;
+        ObjectDoubleMap<String> map = this.company.getCustomers().sumByDouble(cityFunction, totalOrderValueFunction);
```

Comment:
Requesting the same formatting change here as well .

Reply:
Done